admin.py

```diff
@@ -1010,25 +1006,28 @@ class Meta:
-  admin.site.register_view('show_completed_bids', name='Show Completed Bids', urlname='show_completed_bids', view=show_completed_bids)
-  admin.site.register_view('process_donations', name='Process Donations', urlname='process_donations', view=process_donations)
-  admin.site.register_view('read_donations', name='Read Donations', urlname='read_donations', view=read_donations)
-  admin.site.register_view('process_prize_submissions', name='Process Prize Submissions', urlname='process_prize_submissions', view=process_prize_submissions)
-  admin.site.register_view('process_pending_bids', name='Process Pending Bids', urlname='process_pending_bids', view=process_pending_bids)
-  admin.site.register_view('search_objects', name='search_objects', urlname='search_objects', view=views.search, visible=False)
-  admin.site.register_view('edit_object', name='edit_object', urlname='edit_object', view=views.edit, visible=False)
-  admin.site.register_view('add_object', name='add_object', urlname='add_object', view=views.add, visible=False)
-  admin.site.register_view('delete_object', name='delete_object', urlname='delete_object', view=views.delete, visible=False)
-  admin.site.register_view('google_flow', name='google_flow', urlname='google_flow', view=google_flow)
-  # Apparently adminplus doesn't allow parameterized URLS (or at least, I'm not clear on how they work...)
-  # -> the problem seems to be in the urls file, perhaps that I just need to edit that?
-  admin.site.register_view('draw_prize', name='draw_prize', urlname='draw_prize', view=views.draw_prize, visible=False)
-except AttributeError:
-  raise ImproperlyConfigured("Couldn't call register_view on admin.site, make sure admin.site = AdminSitePlus() in urls.py")
+old_get_urls = admin.site.get_urls
+
+def get_urls():
+  urls = old_get_urls()
+  return patterns('',
+                  url('select_event', select_event, name='select_event'),
+                  url('merge_bids', merge_bids_view, name='merge_bids'),
+                  url('merge_donors', merge_donors_view, name='merge_donors'),
+                  url('automail_prize_contributors', automail_prize_contributors, name='automail_prize_contributors'),
+                  url('draw_prize_winners', draw_prize_winners, name='draw_prize_winners'),
+                  url('automail_prize_winners', automail_prize_contributors, name='automail_prize_winners'),
+                  url('show_completed_bids', show_completed_bids, name='show_completed_bids'),
+                  url('process_donations', process_donations, name='process_donations'),
+                  url('read_donations', read_donations, name='read_donations'),
+                  url('process_prize_submissions', process_prize_submissions, name='process_prize_submissions'),
+                  url('process_pending_bids', process_pending_bids, name='process_pending_bids'),
+                  url('search_objects', views.search, name='search_object'),
```

Comment:
`name='search_object'` => `name='search_objects'` (added 's').

(Or at the very least the name should match the url and not differ by pluralization, I can see non-matching names in this case causing really stupid bugs down the road.).